generator-fraudulentlogic/generators/templates/hmrc/README.md

```diff
@@ -0,0 +1,15 @@
+Tax Authority in UK (HMRC) requires software providers using some of their APIs (Eg: MTD VAT API) to provide consistent headers known as Fraud Prevention headers as part of the request. These are HTTP headers that help HMRC to monitor and audit transactions to protect taxpayers’ data from fraudulent activities. 
```

Comment:
I'd probably just give a skeleton outline for the readme: title, description, usage maybe?